.changeset/nine-islands-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+'vite-plugin-checker': minor
```

Comment:
Oh, found this should be `patch` as the version is still lower than 1.0.0. So features will be shipped in patch version.

Reply:
Is it ok if I just manually edit `.changeset/nine-islands-punch.md` to say `patch` or do I need to delete that file an rerun `changeset`?

Reply:
Delete `.changeset/nine-islands-punch.md` and rerun changeset would be fine.

Reply:
Updated